doc/examples/reconst_dki.py

```diff
@@ -228,16 +186,61 @@
    **Diffusion tensor measures obtain from the diffusion tensor estimated from
    DKI (upper panels) and DTI (lower panels).**.
 
-From the figure, we can see that the DT standard diffusion measures from DKI
-are noisier than the DTI measurements. This is a well known pitfall of DKI
-[NetoHe2014]_. Since it involves the estimation of a larger number of
-parameters, DKI is more sensitive to noise than DTI. Nevertheless, DT diffusion
-based measures were shown to have better precision (i.e. less sensitive to
-bias) [Veraa2011]_.
+From the figure, we can see that the standard diffusion measures from DKI has
+similar constrasts than the standard diffusion measures from DTI.
+
+In addition to the standard diffusion statistics, the DiffusinKurtosisFit
+instance can be used to estimate standard non-Gaussian measures as the mean
+kurtosis (MK), the axial kurtosis (AK) and the radial kurtosis (RK).
+"""
+
+MK = dkifit.mk
+AK = dkifit.ak
+RK = dkifit.rk
+
+"""
+Kurtosis measures are susceptible to high amplitude outliers which can corrupt
+the automatic ajusted scale of the matplotlib functions. By assuming that
+typical values of kurtosis lie on a range between 0 and 2, the impact of high
+amplitude kurtosis outliers are removed in the following:
+"""
+
+MK[MK > 2] = 2
+MK[MK < 0] = 0
+AK[AK > 2] = 2
+AK[AK < 0] = 0
+RK[RK > 2] = 2
+RK[RK < 0] = 0
+
+"""
+Now we are ready to plot the kurtosis standard measures using matplotlib:
+"""
 
-The standard kurtosis statistics can be computed from the DiffusinKurtosisFit
-instance as the mean kurtosis (MK), the axial kurtosis (AD) and the radial
-kurtosis (RK).
+fig2, ax = plt.subplots(1, 3, figsize=(12, 6),
+                        subplot_kw={'xticks': [], 'yticks': []})
+
+fig2.subplots_adjust(hspace=0.3, wspace=0.05)
+
+ax.flat[0].imshow(MK[:, :, axial_slice], cmap='gray')
+ax.flat[0].set_title('MK')
+ax.flat[1].imshow(AK[:, :, axial_slice], cmap='gray')
+ax.flat[1].set_title('AK')
+ax.flat[2].imshow(RK[:, :, axial_slice], cmap='gray')
+ax.flat[2].set_title('RK')
+
+plt.show()
+fig2.savefig('Kurtosis_tensor_standard_measures.png')
+
+"""
+.. figure:: Kurtosis_standard_measures.png
```

Comment:
Whoops. Looks like the file name changed here from just a couple of lines before. This will fail to render!